sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.F

```diff
@@ -562,16 +456,25 @@ SUBROUTINE TERSUB(IMN,JMN,IM,JM,NM,NR,NF0,NF1,NW,EFAC,BLAT,
      & ' Before GICE ZAVG(1,52)=',ZAVG(1,52),ZSLM(1,52)
       print *,
      & ' Before GICE ZAVG(1,112)=',ZAVG(1,JMN-112),ZSLM(1,112)
-! GICE: Grumbine 30" Antarctica orog IMNx3616 from S to N & wraped E-W.
-! NB: Zfields are S to N and W-E!
-       iosg = 0
-       READ(15,iostat=iosg) GICE
-       if(iosg .ne. 0 ) then
+
+! Read 30-sec Antarctica RAMP data. Points scan from South
+! to North, and from Greenwich to Greenwich.
+
+      iosg = 0
+      error=NF__OPEN("./fort.15",NF_NOWRITE,fsize,ncid)
```

Comment:
why not just read the actual file name? using fort.15 maintains a level of uncertainty in the data source

Reply:
Done at 2ae071c.